### PR TITLE
Private/mmeeks/simd

### PR DESCRIPTION
**Pull request by mmeeks**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by mmeeks** [comment]
@caolanm Measured - and it appears to (only) double the speed of the RLE code - I was hoping for more, but 2x is at least reasonable... can you review and give it some testing ?
Interestingly - if you compile with -O0 we are slower with AVX2, but at -O2 we are 2x faster - perhaps that comes from pre-optimizing the C version ;-) Thanks !

**Comment by caolanm** [comment]
we're missing DeltaSimd.h, presumably needs a git add for that

File: Makefile.am [hunk 1]
```diff
@@ -115,6 +115,7 @@ shared_sources = common/FileUtil.cpp \
                  common/ConfigUtil.cpp \
                  common/Authorization.cpp \
                  common/CommandControl.cpp \
+		 common/Simd.cpp \
```
Comment (hfiguiere) [inline Makefile.am:118]: 'nit: indentation. My OCD screams here. Also line 319.

**Review by caolanm — APPROVED** [review]
passes ci now

**Pull request merged**